pom.xml

```diff
@@ -108,6 +110,12 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-network-store-client</artifactId>
+            <version>${powsybl-network-store.version}</version>
```

Comment:
<!-- FIXME: to be removed at next powsybl-ws-dependencies upgrade  --> ?

Reply:
Done